### PR TITLE
issue/hitide-ui-48: help catch collection resolution errors

### PR DESCRIPTION
**Pull request by jbyrne6**
### Changes
- Trying to catch cases of resolution being "Not Applicable" to a collection vs when there is an error and the resolution is "Not Available".
- Need to try and find a collection where an error occurs so I can test the error handling.
- Fixed when Image Not Found thumbnail was not showing up correctly.
- Removed Jenkins and Docker files because they are not used anymore in HiTIDE-UI

### Related Issues
- #48 
- #52 
- #49

### CODE REVIEW

**Comment by frankinspace** [comment]
So, the logic here is:

if the cmr key `[Collection]/SpatialExtent/HorizontalSpatialDomain/ResolutionAndCoordinateSystem` exists, look for the `GenericResolutions` key. If `GenericResolutions` is found, use that information to render the resolution label in the UI. If `GenericResoultions` key does not exists, then that is an unexpected error and we render "Issue Loading Resolution" in the UI.

if the cmr key `[Collection]/SpatialExtent/HorizontalSpatialDomain/ResolutionAndCoordinateSystem` does not exist: assume this was intentionally left out, render "Not Applicable" for the resolution label in the UI.

I think that makes sense to me but would appreciate @jjmcnelis opinion as well.

File: src/jpl/utils/SearchDatasets.js [hunk 1]
```diff
@@ -133,19 +134,26 @@ define([
...
+                    // datasetObject["Dataset-Resolution"] = []
                     resolutionObjects.forEach(function(resolutionObject) {
                         var acrossTrack = resolutionObject.XDimension
                         var alongTrack = resolutionObject.YDimension
                         var unit = resolutionObject.Unit
-                        datasetObject["Dataset-Resolution"].push({"Dataset-AcrossTrackResolution": acrossTrack, "Dataset-AlongTrackResolution": alongTrack, "Unit": unit})
+                        datasetObject["Dataset-Resolution"].push({"Dataset-AcrossTrackResolution": acrossTrack, "Dataset-AlongTrackResolution": alongTrack, "Unit": unit, "error": "No Error"})
                     });
+                } else {
+                    // resolution not applicable
```
[the remark this answers is not in the document]
Comment (jbyrne6) [reply src/jpl/utils/SearchDatasets.js:151]: Yes you are correct. This comment was added before I changed made a change to the code.

**Review by frankinspace — APPROVED** [review]

**Review by jamesfwood — APPROVED** [review]

**Comment by jjmcnelis** [comment]
> So, the logic here is:
> 
> if the cmr key `[Collection]/SpatialExtent/HorizontalSpatialDomain/ResolutionAndCoordinateSystem` exists, look for the `GenericResolutions` key. If `GenericResolutions` is found, use that information to render the resolution label in the UI. If `GenericResoultions` key does not exists, then that is an unexpected error and we render "Issue Loading Resolution" in the UI.
> 
> if the cmr key `[Collection]/SpatialExtent/HorizontalSpatialDomain/ResolutionAndCoordinateSystem` does not exist: assume this was intentionally left out, render "Not Applicable" for the resolution label in the UI.
> 
> I think that makes sense to me but would appreciate @jjmcnelis opinion as well.

@frankinspace and @jbyrne6 -- Thanks for the opportunity to review. This logic described in your comment is appropriate, IMO. (So the only time we should see "Issue Loading Resolution" is in the rare case where a dataset has an alternative representation is given under HorizontalDataResolution subfield, e.g. https://cmr.earthdata.nasa.gov/search/collections.umm_json?provider=POCLOUD&ShortName=SEAGLIDER_GUAM_2019 and datasets with that characteristic are not likely to be in supported in hitide.)

**Review by sliu008 — CHANGES_REQUESTED** [review]
we still need the read_collection.py file

**Pull request merged**